### PR TITLE
Add Breadcrumb component

### LINKED ISSUE #11: Component - breadcrumb
image

### PR DESCRIPTION
**Pull request by BrookJeynes**
Closes #11

image
**Figure: New breadcrumb component**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -16,3 +16,15 @@
   @import "react-responsive-carousel/lib/styles/carousel.min.css";
 }
 
+.breadcrumb_item {
+    @apply inline text-[0.8rem] text-[#666];
+}
+
+.breadcrumb_item a {
+    @apply no-underline;
+}
```
Comment (tkapa) [inline styles.css:25]: Move these our of styles.css. Use Tailwind classes as much as possible.

File: tailwind.config.js [hunk 2]
```diff
@@ -89,6 +89,9 @@ module.exports = {
         7: ".075",
         15: ".15",
       },
+      width: {
+        224: "80rem",
+      },
```
Comment (tkapa) [inline tailwind.config.js:94]: Try using the same classNames as the body content

**Review by tkapa — CHANGES_REQUESTED** [review]
Looks good, some comments to take a look at.

File: styles.css (hunk 1 above)
Comment (wicksipedia) [reply styles.css:25]: Yeah these are specific to breadcrumbs only, can we pull the styles next to the component (even better if it was in the component)

**Review by bradystroud — APPROVED** [review]
⛴️ LGTM

**Review by tkapa — DISMISSED** [review]

**Review by tkapa — APPROVED** [review]

**Pull request merged**